templates/chatpad/meta.yaml

```diff
@@ -0,0 +1,32 @@
+name: Chatpad
+description: Not just another ChatGPT user-interface!
+instructions: null
+changeLog:
+  - date: 2023-3-26
+    description: first release
+links:
+  - label: Website
+    url: https://chatpad.ai
+  - label: Github
+    url: https://github.com/deiucanta/chatpad
+contributors:
+  - name: Supernova3339
+    url: https://github.com/Supernova3339
+schema:
+  type: object
+  required:
+    - projectName
+    - appServiceName
+    - appServiceImage
+  properties:
+    projectName:
+      type: string
+      title: Project Name
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: chatpad
+    appServiceImage:
+      type: string
+      title: App Service Image
+      default: ghcr.io/deiucanta/chatpad:b29ca06fb220ce74b5b5a051d414ec82fcdb2829
```

Comment:
in this case we can use latest since new versions cannot break anything - there is no db connection, there are no mounts, no ports, so nothing to break

Reply:
Agreed, since the data is stored locally in the browsers storage.